integrationtests/server-integration/third-party-server/src/test/java/org/infinispan/test/integration/DeploymentHelper.java

```diff
@@ -0,0 +1,57 @@
+package org.infinispan.test.integration;
+
+import java.io.File;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.jboss.shrinkwrap.resolver.api.maven.MavenResolverSystem;
+
+public class DeploymentHelper {
+
+   public static WebArchive createDeployment() {
+
+      WebArchive war = ShrinkWrap.create(WebArchive.class, "infinispan-server-integration.war");
+      String arquillianLaunch = System.getProperty("arquillian.launch");
+
+      if (arquillianLaunch.equals("tomcat")) {
+         tomcat(war);
+      } else if (arquillianLaunch.equals("wildfly")) {
+         wildfly(war);
+      } else {
+         throw new IllegalStateException(String.format("%s not supported", arquillianLaunch));
+      }
+
+      MavenResolverSystem resolver = Maven.resolver();
+      addLibrary(war, resolver
+            .loadPomFromFile("pom.xml")
+            .resolve("org.infinispan:infinispan-client-hotrod")
+            .withTransitivity().as(File.class));
+      return war;
+   }
+
+   private static void wildfly(WebArchive war) {
+      String jbossDeploymentStructure = "<jboss-deployment-structure xmlns=\"urn:jboss:deployment-structure:1.1\">\n" +
```

Comment:
IMO if we have jboss-deployment-structure.xml inside `src/test/resources/eap/jboss-deployment-structure.xml`is way better than work with string into the code